assets/app/scripts/services/builds.js

```diff
@@ -102,5 +102,37 @@ angular.module("openshiftConsole")
       return buildConfigBuildsInProgress;
     };
 
+    function isPaused(buildConfig) {
+      if ($filter('annotation')(buildConfig, "openshift.io/build-config.paused") === 'true') {
+        return true;
+      }
+      return false;
+    }
```

Comment:
Just inline the implement in the `BuildService.prototype.isPaused` method below and call `this.isPaused(buildConfig)` in the `canBuild` method below.